FlowersStore/Scenes/FlowersList/FlowersListPresenter.swift

```diff
@@ -20,16 +20,26 @@ final class FlowersListPresenter {
     private weak var output: FlowersListPresenterOutput?
     private let interactor: FlowersListInteractorInput
```

Comment:
<!--
  0 failure: 
  1 warning:  Property 'interac...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/Scenes/FlowersList/FlowersListPresenter.swift;
  Line: 21;
-->

- :warning: Property 'interactor' is assigned, but never used